src/app/zap-templates/templates/app/im-cluster-command-handler.zapt

```diff
@@ -39,6 +39,7 @@ void DispatchServerCommand(CommandHandler * apCommandObj, const ConcreteCommandP
         {
     {{/first}}
         case Commands::{{asUpperCamelCase commandName}}::Id: {
+            MatterPreExecuteCommandCallback(apCommandObj, aCommandPath);
```

Comment:
Why is this not done 1 time before DispatchServerCommand?

Reply:
Because MatterPreExecuteCommandCallback(apCommandObj, aCommandPath) can be called for unknown command or unknown cluster. For this case works "default" section in "case". And user get notification (callback was called) about command execution